kolibri/plugins/user/assets/src/views/profile-page/index.vue

```diff
@@ -143,15 +149,23 @@
         }
         return true;
       },
-      usernameIsAlphaNumUnderscore() {
-        return /^\w+$/g.test(this.username);
+      nameIsInvalidText() {
+        if (this.nameBlurred || this.formSubmitted) {
+          if (this.name === '') {
+            return this.$tr('required');
+          }
+        }
+        return '';
+      },
+      nameIsInvalid() {
+        return !!this.nameIsInvalidText;
       },
       usernameIsInvalidText() {
-        if (this.usernameBlurred || this.validateForm) {
+        if (this.usernameBlurred || this.formSubmitted) {
           if (this.username === '') {
```

Comment:
could just move the condition for this if into the if above it

Reply:
I could, but I was just following the pattern where the first if checks if validation should occur, and then inside there is where validation occurs.